ncj/3dsmax/vray-dr/job.template.json

```diff
@@ -199,14 +216,18 @@
                         },
                         "multiInstanceSettings":{
                             "numberOfInstances":"[parameters('nodeCount')]",
-                            "coordinationCommandLine":"cmd.exe /c if \"[parameters('vrayRenderer')]\" == \"VRayAdv\" (cmd.exe /c start \"\" \"C:\\Autodesk\\3dsMax2018\\vrayspawner2018.exe\" \"-port=[parameters('vrayPort')]\") else (cmd.exe /c start C:\\server\\3dsmaxVray\\bin\\vray.exe -server -portNumber=[parameters('vrayPort')])",
+                            "coordinationCommandLine":"cmd /c %AZ_BATCH_JOB_PREP_WORKING_DIR%/PrepareVrayServices.bat [parameters('maxVersion')] [parameters('vrayRenderer')] [parameters('vrayPort')])",
```

Comment:
These args aren't in the correct order.  The script takes <VRay> <Max> <Port>.  I assume this hasn't been tested, can you test the changes before you merge please?

Reply:
Updated the bat file 👍